File: fixtures/python_output/post_binary_file.py
import requests

headers = {
    'Content-type': 'application/sparql-query',
    'Accept': 'application/sparql-results+json',
}

data = open('./sample.sparql', 'rb').read()
requests.post('http://lodstories.isi.edu:3030/american-art/query', headers=headers, data=data)

File: fixtures/elixir_output/post_binary_file.ex
method = :post
url = "http://lodstories.isi.edu:3030/american-art/query"
headers = [
  {~s|Content-type|, ~s|application/sparql-query|},
  {~s|Accept|, ~s|application/sparql-results+json|},
]
body = File.read!("./sample.sparql")
options = []
params = []

request = %HTTPoison.Request{
  method: method,
  url: url,
  body: body,
  headers: headers,
  options: options,
  params: params,
}

response = HTTPoison.request(request)
